src/ProjectWindow.cpp

```diff
@@ -39,6 +39,7 @@ Paul Licameli split from AudacityProject.cpp
 #include <wx/display.h>
 #include <wx/scrolbar.h>
 #include <wx/sizer.h>
+#include <wx/splitter.h>
```

Comment:
The only new #include in this PR, and it's a wxWidgets header.  No new dependencies among Audacity's own source code files.  That is good.